ocs_ci/templates/ocs-deployment/multicluster/submariner_downstream_unreleased_catsrc.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: submariner-catalogsource
+  namespace: openshift-marketplace
+spec:
+  icon:
+    base64data: ""
+    mediatype: ""
+  image: brew.registry.redhat.io/rh-osbs/iib:PLACE_HOLDER
```

Probably format it as a whole string outside and here you can just have PLACEHOLDER
```suggestion
  image: PLACE_HOLDER
```

---

done

---

done